edward/optimizers/sgd.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+
+class KucukelbirOptimizer:
+
+  """
+  Used for RSVI (Rejection-Sampling Variational Inference).
+
+  # TODO: add me
+  """
+
+  def __init__(self, t, delta, eta, s_n, n):
+    self.t = t
+    self.delta = delta
+    self.eta = eta
+    self.s_n = s_n
+    self.n = n
+
+  def apply_gradients(self, grads_and_vars, global_step=None):
```

Comment:
@dustinvtran 

I'd quite appreciate if you could glance at this method as well, as my integration test passes on some days and fails on others — with 0 changes to my code. Promise 🤞.